docs/editors-guide/rare-disease-curator.md

```diff
@@ -0,0 +1,51 @@
+# Rare Disease Curation
+
+Rare diseases in Mondo are created based on 3 main subsets. Rules for editing these subsets depends on the subset type. Before editing rare disease information in Mondo, please familiarize yourself with how rare diseases are represented in Mondo [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-subset/).
+
+It is important to remember that the rare disease subsets are updated automatically at the time of the monthly Mondo release. It is, therefore, crucial that editing rare disease information is done by following the instructions below for these edits to be retained after the release.
+
```

Comment:
@matentzn is that true? Or should curators run something like 
sh run.sh make mondo.owl MIR=false IMP=false
?

Reply:
The sentence here is correct. A curator that really wants to rebuild the rare subset (the owl file) can run:

```
sh run.sh make rare_subset
```